src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/PrisonerServiceTest.kt

```diff
@@ -110,7 +110,7 @@ class PrisonerServiceTest {
 
     val mockedJsonResponse: PrisonersSearchList = readFileAsObject("testdata/prisoner-search-api/prisoner-search-1.json")
     `when`(prisonerSearchApiService.findPrisonerPersonalDetails(expectedPrisonerId)).thenReturn(mockedJsonResponse.content!![0])
-    `when`(pathwayAndStatusService.getOrCreatePrisoner(eq(expectedPrisonerId), any(), any())).thenReturn(mockEntity)
+    `when`(pathwayAndStatusService.getOrCreatePrisoner(expectedPrisonerId, "MDI", LocalDate.parse("2099-09-12"), null)).thenReturn(mockEntity)
```

Comment:
I tried just putting another `any()` in but mockito was not having it for some reason, removing the matchers fixed it 🤷

Reply:
mockito doesn't like optional null params